publiccode.yml

```diff
@@ -21,113 +15,67 @@ outputTypes:
 platforms:
   - GNU/Linux
 
-# For a list of allowed values, check out w3id.org/publiccode/version/0.1/tags.html
-#
-# Only stick to these tags! If you need additional tags (localised),
-# you have the chance of adding them under the description/freeTags key
-tags:
-  - concorsi personale tecnico amministrativo e docenti
-  - bandi
+categories:
+  - procurement
+  - hr
 
 usedBy:
   - Università della Calabria
 
-# roadmap: "https://github.com/italia/django-peo/roadmap"
-
 developmentStatus: stable
-softwareType: "standalone"
-
+softwareType: standalone/web
 
 description:
-  eng:
+  en:
     localisedName: Peo
     genericName: Django peo
-    shortDescription: >
+    shortDescription: |- 
           Gestione, redazione e pubblicazione Bandi e Concorsi per
           personale Tecnico Amministrativo e personale Docente.
 
-    longDescription: >
+    longDescription: |- 
           Gestione, redazione e pubblicazione Bandi e Concorsi per
           personale Tecnico Amministrativo e personale Docente.
 
     documentation: "https://peo.unical.it/docs"
-    apiDocumentation: "https://docs.djangoproject.com/en/2.1/"
 
-    featureList:
+    features:
        - Integrazione in Cineca CSA (customizzabile o standalone)
        - Integrazione con protocollo informatico WSArchiPRO (customizzabile)
        - Autenticazione SAML2
-    # screenshots:
-       # - img_git/sshot1.jpg
-       # - img_git/sshot2.jpg
-       # - img_git/sshot3.jpg
-    # videos:
-       # - https://youtube.com/xxxxxxxx
-    # awards:
-       # - 1st Price Software of the year
+    screenshots:
+       - data/gallery/home.png
+       - data/gallery/domanda1.png
+       - data/gallery/modulo.png
+       - data/gallery/costruzione_modulo.png
 
 legal:
-  license: GPL-3.0-or-later        # SPDX expression of license
+  license: GPL-3.0-or-later
   mainCopyrightOwner: Università della Calabria
   repoOwner: Università della Calabria
   authorsFile: AUTHORS
 
 maintenance:
-  type: "contract"
-
-  contractors:
-    - name: "Fornitore Privato SPA"
-      website: "https://privatecompany.com"
-      until: "2019-01-01"
+  type: "internal"
 
   contacts:
```

Comment:
Seeing the standard, this should be `Questa chiave contiene il nome completo di uno dei contatti tecnici. Deve essere una persona reale; NON popolare questa chiave con informazioni di contatto generiche, dipartimenti dell’azienda, associazioni, etc.` https://docs.italia.it/italia/developers-italia/publiccodeyml/it/master/schema.core.html#contatto